src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -121,6 +121,9 @@ def __init__(self, jobID: int | None = None, jobReport: JobReport | None = None)
         self.pilotRef = gConfig.getValue("/LocalSite/PilotReference", "Unknown")
         self.cpuNormalizationFactor = gConfig.getValue("/LocalSite/CPUNormalizationFactor", 0.0)
         self.bufferLimit = gConfig.getValue(self.section + "/BufferLimit", 10485760)
+        logLevel = gConfig.getValue(self.section + "/LogLevel", "VERBOSE")
```

Comment:
Is "VERBOSE" already the default?

Reply:
Wait, it sounds like we can actually modify the `LogLevel` of the `JobWrapper` by either:
- changing the log level in the JDL of a given job
- or changing the log level of the `JobAgent`

I will give it a try